section2/task2/README.md

```diff
@@ -27,12 +27,13 @@ import datetime
 # all third-party libraries in the middle
 import backtrader as bt
 import pandas as pd
+import matplotlib as plt
```

Comment:
I think both of them work. I can change it and pull again I guess.

Reply:
Not sure if it's an upgrade of matplotlib. But since all [examples](https://matplotlib.org/gallery/index.html) from its doc use `import matplotlib.pyplot as plt`. I would prefer go with that to stay safe.